features/step_definitions/advocate_claim_steps.rb

```diff
@@ -40,7 +40,8 @@
 
 Then(/^I select the first search result$/) do
   sleep Capybara.default_max_wait_time
-  @claim_form_page.offence_results.first.select_button(visible: false).trigger('click')
+  find(:xpath, '//*[@id="offence-list"]/div[3]/div').hover
```

Comment:
As discussed, can we just put a `# TODO: get working in site-prism` on this.

Reply:
I've added an issue, #2454